examples/viewer.py

```diff
@@ -0,0 +1,39 @@
+from magnum import gl
+from magnum.platform.glfw import Application
+from settings import default_sim_settings, make_cfg
+
+import habitat_sim
+
+# Setting filepath to load
+sim_settings = default_sim_settings
+sim_settings[
+    "scene"
+] = "data/scene_datasets/habitat-test-scenes/frl_apartment_stage.stage_config.json"
+
+
+class SkeletonPythonViewer(Application):
+    def __init__(self):
+        configuration = self.Configuration()
+        configuration.title = "Skeleton Viewer Application"
+        Application.__init__(self, configuration)
+
+        self.viewport_size = gl.default_framebuffer.viewport.size()
+
+        self.cfg = make_cfg(sim_settings)
```

Comment:
The correct line may be different in Harsh's branch, since the bindings have changed: what was "scene.id" is now "scene_id".